src/angular-google-analytics.js

```diff
@@ -353,10 +353,24 @@
             }
             if (angular.isArray(accountId)) {
               accountId.forEach(function (trackerObj) {
-                $window.ga(_generateCommandName('send', trackerObj), 'pageview', opt_fieldObject);
+                //delay because it might be that custom dimension are set in the angular app after the event gets fired.
+                //the event gets fired on a very early state of the angular app.
+                //Example from googlr documentation
```

Comment:
Spelling mistake here. The convention in this project is that comments have a space after the '//'.